### PR TITLE
[Chore]: Technical: fix linting errors of @types/styled-components plugin

### PR DESCRIPTION
**Pull request by jagerts**
Signed-off-by: Evgeny Zhgulev <evgeny.zhgulev@actionengine.com>

### CODE REVIEW

File: src/components/common/file-uploader/file-upload-progress.tsx [hunk 1]
```diff
@@ -25,11 +25,9 @@ import {TruncatedTitleText} from 'components/common/styled-components';
 import {getError} from 'utils/utils';
 import {FileLoadingProgress} from 'reducers/vis-state-updaters';
 
-/** @typedef {import('./file-upload-progress').FileUploadProgressProps} FileUploadProgressProps*/
-
-const StyledFileProgress = styled.div.attrs({
-  className: 'file-upload__progress'
-})`
+const StyledFileProgress = styled.div.attrs(props => ({
+  className: `file-upload__progress${props.className ? ` ${props.className}` : ''}`
```
Comment (heshan0131) [inline src/components/common/file-uploader/file-upload-progress.tsx:29]: use `classnames` here?

File: src/components/common/dataset-label.tsx [hunk 2]
```diff
@@ -31,15 +32,12 @@ const DatasetName = styled.div.attrs({
 `;
 
 interface DatasetLabelType {
-  dataset: {
-    color?: string;
-    label?: string;
-  };
+  dataset: KeplerTable;
```
Comment (heshan0131) [inline src/components/common/dataset-label.tsx:35]: can we keep the original minified type?

Comment (jagerts) [reply src/components/common/dataset-label.tsx]: yes, we can. Fixed

File: src/components/common/file-uploader/file-upload-progress.tsx (hunk 1 above)
Comment (jagerts) [reply src/components/common/file-uploader/file-upload-progress.tsx]: fixed

File: src/components/common/portaled.tsx [hunk 3]
```diff
@@ -140,10 +140,11 @@ interface PortaledProps {
   isOpened?: boolean;
   top: number;
   left?: number;
-  right: number;
+  right?: number;
   overlayZIndex?: number;
   modalProps?: Partial<ReactModal.Props>;
   modalStyle?: Partial<typeof defaultModalStyle>;
+  children: React.ReactNode;
```
Comment (heshan0131) [inline src/components/common/portaled.tsx:147]: shouldn't need to add children here

**Review by heshan0131 — APPROVED** [review]

File: src/components/common/portaled.tsx (hunk 3 above)
Comment (jagerts) [reply src/components/common/portaled.tsx]: fixed

**Comment by jagerts** [comment]
@heshan0131 i've merged all PR's about styled-components. So i added @types/styled-components and resolved some errors left in this pr. Check changes out, plz.

**Pull request merged**